renovate.json

```diff
@@ -24,6 +24,10 @@
       {
         "packagePatterns": ["^eslint"],
         "groupName": "eslint packages"
+      },
+      {
+       "matchPackageNames": ["psycopg2-binary"],
+       "allowedVersions": "<2.9"
```

Comment:
Do we need this? Is it also js and shouldn't be updated by renovate?

Reply:
I'm not 100% sure to be honest, but on bplan renovate had a MR for it, renovate does python dependencies ? or at least sometimes it does, there doesn't seem to be a clear line

Reply:
hmm, I guess you are right, maybe it was a bug in bplan, all the other repos only get js updates via renovate. I'll remove this.